include/NovelRT/Maths/GeoVector3.h

```diff
@@ -0,0 +1,208 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_MATHS_GEOVECTOR3_H
+#define NOVELRT_MATHS_GEOVECTOR3_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Maths {
+  template<typename T>
+  class GeoVector3 {
+    friend class Graphics::RenderObject;
```

Comment:
Are all of these friends valid?

Reply:
Cleaned up all friend classes for the vector types to only what's being used.